ext/websocket/lib.rs

```diff
@@ -504,18 +514,46 @@ pub async fn op_ws_close(
   Ok(())
 }
 
+#[op(fast)]
+pub fn op_ws_get_buffer(state: &mut OpState, rid: ResourceId) -> ZeroCopyBuf {
```

Comment:
`#[op(fast)]` is failing to actually validate fastness... These ops are not fast-compliant.

That being said, maybe I should create a way in Fast API to return new buffers...

Reply:
@aapoalas do you happen to know if there's an issue about it?

Reply:
Not quite sure. Maybe not, though I might have created one...